charmcraft.yaml

```diff
@@ -0,0 +1,43 @@
+# This file configures Charmcraft.
+# See https://juju.is/docs/sdk/charmcraft-config for guidance.
+
+name: gocert-k8s
+type: charm
+title: GoCert
+summary: charming certificate management made easy
+
+description: |
+  GoCert is an application that helps you manage certificate requests and their associated certificates.
+  Charmed GoCert can help you automatically receive CSRs and distribute certificates to the applications
+  you've deployed in your model.
+
+provides:
+  certificates:
+    interface: tls-certificates
+
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
+
+containers:
+  gocert:
+    resource: gocert-image
+
```

Comment:
We should have persistent storage for our database, and likely for the directory that will hold the configuration file

Reply:
I think using storage mounts sense to do in a separate PR, it has some edge cases like handling mounting and unmounting that should be reviewed in detail.